example/torus.red

```diff
@@ -21,7 +21,7 @@ let t2c (t : torus) : s1 × s1 =
   | pt ⇒ <base, base>
   | p/one i ⇒ <loop i, base>
   | p/two i ⇒ <base, loop i>
-  | square i j ⇒ refl
```

Comment:
:laughing:

Reply:
taste it

Reply:
It's a good test, but it is a little confusing in context.